README.md

```diff
@@ -7,31 +7,56 @@ Setup the repo
 ```bash
 git clone git@github.com:aaa-nyc-hackathon/aaa-nyc-llama-hackathon.git
```

Comment:
the only file needing review is this `README.md` file. The other changes files are the result of executing the `ruff` commands for auto-linting and auto-formatting described in this file.